cli/cmd/component-delete.go

```diff
@@ -63,14 +63,24 @@ func runDelete(cmd *cobra.Command, args []string) {
 		log.SetLevel(log.DebugLevel)
 	}
 
-	if err := componentDelete(contextLogger, args); err != nil {
+	options := componentDeleteOptions{
+		confirm:         confirm,
+		deleteNamespace: deleteNamespace,
+	}
+
+	if err := componentDelete(contextLogger, args, options); err != nil {
```

Comment:
why not also add args [components to delete] in the options struct ?

Reply:
Hm, I imagine they act on sort of different levels. `args` and `contextLogger` as invokation-specific, while options struct could possibly be re-used between multiple `componentDelete()` calls.